R/whatWQPdata.R

```diff
@@ -22,7 +22,7 @@ whatWQPsamples <- function(...){
                     urlCall,
                     "&mimeType=tsv")
   
-  retval <- importWQP(urlCall, zip=values["zip"] == "yes")
+  retval <- suppressWarnings(importWQP(urlCall, zip=values["zip"] == "yes"))
```

Comment:
I think there was something in particular here generating an innocuous warning, right?  Is there any way you could more surgically handle that warning without suppressing others?

Reply:
yeah, every R update or so I cross my fingers that they'll just add that option to suppressWarning for a specific regex.... But, I guess the language translation issue would make that impossible.  `suppressWarning` is one of those things I use the lazy solution because the `withCallingHandlers`+`invokeRestart` makes me more nervous that I'll screw it up.  The other option of course is to leave it in. Jim says they will fix it eventually....

Reply:
I'd say leave the warning, or make an issue to document why the `suppressWarning` is there/when it can be removed.